src/options.ts

```diff
@@ -81,6 +81,19 @@ export const getSourcemaps = (): string[] => {
   return sourcemapsOption.split(' ');
 };
 
+/**
+ * Dist is optional, but should be a string when provided.
+ * @returns string
+ */
+export const getDist = (): string | undefined => {
+  const distOption: string = core.getInput('dist');
+  if (!distOption) {
+    return undefined;
```

Comment:
this should probably return `null` not undefined, as it looks like other getters use `null`, eg the `getStartedAt`.

Will make this change.

Reply:
```suggestion
    return null;
```